### DISCUSSION TITLE
View URL in Product throws error

### DISCUSSION THREAD

**Discussion by Griehle** [duncanmcclean/simple-commerce#825] [Help] [answered]
If I go into statamic cp > Products > Adyghe cheese and click the view URL I get an "Unmatched ')' on line 6 of: {{ ratio = null }} {{ ratio_large = null }}"  error. Is this because of my failure to add something or is it actually jacked up?

**Comment by duncanmcclean** [comment, marked as answer]
That sounds like an Antlers error. Have you made any changes to any of the templates?

**Reply by Griehle** [reply]
Not yet. Its hard to make heads or tails of the documentation. I have created a taxonomy,  added simple commerce, and added product entries but as yet have been unable to view the pages. I just did scaffold index and show methods. Its kinda hard to figure this out. Its so different from anything I have worked with in Laravel.

**Reply by duncanmcclean** [reply]
Are you using the Antlers Regex parser? If you are, switch to `runtime` as that's what the Starter Kit templates use (they use a syntax that breaks in the old regex parser).

You can change the parser used in your `config/statamic/antlers.php` config file. [Read more on the Statamic Docs](https://statamic.dev/antlers#content).

**Reply by Griehle** [reply]
This was the solution I needed. Thank you.
